hathor/indexes/manager.py

```diff
@@ -63,6 +64,22 @@ def enable_tokens_index(self) -> None:
         """Enable tokens index. It does nothing if it has already been enabled."""
         raise NotImplementedError
 
+    def _manually_initialize(self, tx_storage: 'TransactionStorage') -> None:
```

Comment:
Maybe rename to `_manually_initialize_tips_index(...)`?

Reply:
Makes sense, I'll rename it to `_manually_initialize_tips_indexes`.

Reply:
Done.